test/e2e/storage/testsuites/provisioning.go

```diff
@@ -173,7 +173,7 @@ type provisioningTestInput struct {
 }
 
 func testProvisioning(input *provisioningTestInput) {
-	It("should provision storage", func() {
+	It("should provision storage without mount options", func() {
```

Comment:
Maybe a better name would be "with default storageclass options". For example there are provisioning test cases for reclaim policy and other fields

Reply:
Should I then also rename the "should provision storage with mount options" test?

Right now it is consistent: https://github.com/kubernetes/kubernetes/blob/9df867dac94c35f1c9b74739a585ec38be911f06/test/e2e/storage/testsuites/provisioning.go#L175-L187

If I rename just this name here, the difference between these two test cases would become less obvious.

Reply:
Storage class options is not the same as mount options.  Maybe the use of options for storage class is not the best.  Storage class settings? Storage class fields?

Reply:
How about "should provision storage with defaults" and "should provision storage with mount options"?

In the end I don't care too much. @msau42 please pick something for both tests and I'll change the PR accordingly.

Reply:
With defaults sounds fine to me. Thanks!

Reply:
@msau42 done